XMLSerializer: Note that the output might not be valid XML (#35585)
mdn/content: files/en-us/web/api/xmlserializer/index.md

The `XMLSerializer` interface provides the {{domxref("XMLSerializer.serializeToString", "serializeToString()")}} method to construct an XML string representing a {{Glossary("DOM")}} tree.

> > [!NOTE]
> > The resulting XML string is not guaranteed to necessarily be valid XML, for instance, if element names or attribute names are not permissible XML names.

Reviewer: ```suggestion
> [!NOTE]
> The resulting XML string is not guaranteed to be well-formed XML.
```
